generic-helpers/src/test/java/io/elimu/a2d2/helpers/JedisHelperTest.java

```diff
@@ -0,0 +1,40 @@
+package io.elimu.a2d2.helpers;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import redis.embedded.RedisServer;
+
+public class JedisHelperTest {
+
+	private static RedisServer redis;
+	
+	@BeforeClass
+	public static void startup() {
+		String host = System.getProperty("cache.jedis.host", "localhost");
+		if ("localhost".equals(host)) {
+			int port = Integer.valueOf(System.getProperty("cache.jedis.port", "6379"));
+			redis = new RedisServer(port);
+			redis.start();
```

Comment:
1. I think we should not run actual redis server for executing Unit tests.
2. In Past, running Redis server for Unit test used to give thread stack alerts.(@hankwallace @vishal-elimu was it on jenkins server)

Reply:
@ddjain changed this to use redis-mock instead, not starting an actual redis server